parm/FV3LAM_wflow.xml

```diff
@@ -1605,10 +1606,79 @@ xml.  This may be changed in the future.
 
 
 {#-
-Obs pre-processing tasks that must be run if either deterministic or
-ensemble verification tasks are going to be run.
+Obs processing tasks that must be run if either deterministic or ensemble
+verification tasks are going to be run.
 #}
 {%- if run_tasks_metvx_det or run_tasks_metvx_ens %}
+
+{#-
+Task for pre-processing of NDAS observations to convert prep-buffer files
+to NetCDF format.
+    {%- set obtype = "NDAS" %}
+#}
+  {%- if ("SFC" in vx_fields) or ("UPA" in vx_fields) %}
+{#-
+This for-loop isn't strictly necessary because it loops over only one 
+item, but having it allows for the use of the "set" tag to define new
+variables within the loop (in Jinja, it is not possible to define variables
+outside a loop unless one uses a namespace, which would be overkill in
+this case).  So here we opt for a redundant for-loop.
+#}
+    {%- for obtype in ["NDAS"] -%}
+      {%- set tn = tn_run_met_pb2nc_obs -%}
+      {%- set maxtries = maxtries_run_met_pb2nc_obs -%}
+      {%- set nnodes = nnodes_run_met_pb2nc_obs -%}
+      {%- set ppn = ppn_run_met_pb2nc_obs -%}
+      {%- set wtime = wtime_run_met_pb2nc_obs -%}
+<!--
+************************************************************************
+************************************************************************
+-->
+  <task name="{{tn}}" cycledefs="forecast" maxtries="{{maxtries}}">
+
+    &RSRV_DEFAULT;
+    <command>&LOAD_MODULES_RUN_TASK_FP; "&VX_LOCAL_MODULE_FN;" "&JOBSdir;/JREGIONAL_RUN_MET_PB2NC_OBS"</command>
+    <nodes>{{nnodes}}:ppn={{ppn}}</nodes>
+    <walltime>{{wtime}}</walltime>
+    <nodesize>&NCORES_PER_NODE;</nodesize>
+    <native>&SCHED_NATIVE_CMD;</native>
+    <jobname>{{tn}}</jobname>
+    <join><cyclestr>&LOGDIR;/{{tn}}_@Y@m@d@H.log</cyclestr></join>
```

Comment:
To make nco mode work
```suggestion
    <join>&LOGDIR;/{{tn}}_<cyclestr>@Y@m@d@H</cyclestr>&LOGEXT;</join>
```

Reply:
@danielabdi-noaa, thanks for pointing this out!

Reply:
@danielabdi-noaa Thanks for pointing this out.  I changed it, but there are other issues in NCO mode that prevent the tasks working in that mode (mostly paths need to be set correctly).  I will address them in my next PR.